src/TesApi.Tests/ContractResolverTests.cs

```diff
@@ -42,7 +42,7 @@ public void BasicContractResolver_ReturnsBasicJsonResult()
             var expectedBasicJsonResult = JObject.Parse(File.ReadAllText("expectedBasicJsonResult.json"));
 
             // Act
-            var jsonResult = JsonConvert.SerializeObject(tesTask, new JsonSerializerSettings { ContractResolver = BasicTesTaskContractResolver.Instance });
+            var jsonResult = JsonConvert.SerializeObject(tesTask, new JsonSerializerSettings { ContractResolver = BasicTesTaskContractResolver.Instance, NullValueHandling = NullValueHandling.Ignore });
```

Comment:
There should be no NullValueHandling = NullValueHandling.Ignore here, as the serialized object here will not look like the one produced by the API. The new properties on the TesResources object should be exposed just like all the other properties there. This means serializing nulls as well. I suggest adding the sample values to the tesTaskExample.json and all three expected*.json files. 
BTW, the custom serialization logic needs to be in contract resolvers. That way the test code and the API code will produce the same result. In this case there is nothing to change there.